### PR TITLE
Update to latest client changes

### PR DESCRIPTION
**Pull request by tortmayr**
#### What it does
- Update glsp next dependencies
- Use debuggable feature ids for all vscode feature modules
- Fix bug with `vscodeNavigationModule` that accidently reused the feature id from the navigation module instead of requiring it
- Adjust tool palette css
- Customize css for newly introduced features
- Ensure that `GLSPProjectionView` is properly rendered
   - disable overflow of root div
   - Add mouse listeners to diagram widget to only show projections bars when the diagram is focused
- Update example1.wf
- Fix title menu contributions. Vscode now requires an icon to render them properly

#### Changelog

-   [ ] This PR should be mentioned in the changelog
-   [ ] This PR introduces a breaking change (if yes, provide more details below for the changelog and the migration guide)

### CODE REVIEW

File: example/workflow/extension/package.json [hunk 1]
```diff
@@ -60,21 +60,24 @@
         "enablement": "activeCustomEditorId == 'workflow.glspDiagram'"
       },
       {
-        "command": "workflow.goToNextNode",
-        "title": "Go to next Node",
+        "command": "workflow.goToPreviousNode",
```
Comment (martin-fleck-at) [inline example/workflow/extension/package.json:63]: For some reason the navigation does not seem to work properly for me:
- The 'go to previous' is never even shown.
- The 'go to right' gets stuck quickly but that might be the expected behavior?
- The 'Show documentation...' does not work even if I have the 'Push' node selected.

Maybe something is off on my side, do you have any idea what that could be?

**Review by martin-fleck-at — CHANGES_REQUESTED** [review]
Change looks good to me and works well besides that I have some issues with the navigation and showing of the documentation. Maybe it is something on my side (see inline comment). As soon as I figure out what that could be, this can be merged. Thank you for the good work, Tobias!

**Comment by tortmayr** [comment]
Thanks for the review
> The 'go to previous' is never even shown.

Fixed 

> The 'go to right' gets stuck quickly but that might be the expected behavior?
I guess you were testing on a fork or decision node. Then this is expected behavior, has the previous/next nav commands only work for single connected edges

>The 'Show documentation...' does not work even if I have the 'Push' node selected.

Nice catch. Should be fixed with https://github.com/eclipse-glsp/glsp-server-node/pull/89

**Review by martin-fleck-at — APPROVED** [review]
I can confirm that it works with the newer server version! Also the navigation now shows up properly. One minor thing that could be improved is the order of the two buttons, it is a bit counter-intuitive:
image

But that should not stop this PR from being merged.

**Comment by tortmayr** [comment]
>One minor thing that could be improved is the order of the two buttons, it is a bit counter-intuitive:
Agreed, but I already wasted an unreasonable amount of time trying to fix that.
The problem is that the normal approach of ordering menu contributions (`group:navigation@1` etc) does not seem to work for the `editor/title` menu.
The order in the package.json also seems to have no effect (previous command is before the next but still rendered after)

I guess the only way to fix this would to change the command names to enforce correct lexicographical ordering. But that would be a breaking change which seems a bit overkill for that.

**Comment by martin-fleck-at** [comment]
Interesting, yeah, then let's leave it as it is for now. Thank you!

**Pull request merged**